### ISSUE TITLE
Running configure multiple times with CFBundleUrlScheme does not merge dicts correctly

### ISSUE THREAD

**Issue by kentora** [ionic-team/capacitor-configure#32]
With yaml file containing
```
       infoPlist:
          - CFBundleUrlTypes:
              - CFBundleUrlSchemes:
                  - MyApp
```
running the configure command multiple times yields the following content in info.plist:
```
    <key>CFBundleUrlTypes</key>
    <array>
      <dict>
        <key>CFBundleUrlSchemes</key>
        <array>
          <string>MyApp</string>
        </array>
      </dict>
      <dict>
        <key>CFBundleUrlSchemes</key>
        <array>
          <string>MyApp</string>
        </array>
      </dict>
    </array>
```

Expected behaviour is to merge correctly, instead of adding an entry on each run.

**Comment by mlynch** [comment]
Which version are you on? Will take a look this week

**Comment by kentora** [comment]
I'm on 1.0.22 :)

**Comment by mlynch** [comment]
Fixed in 1.0.24, thanks!
